spec/controllers/account/users_controller_spec.rb

```diff
@@ -71,14 +62,25 @@
     end
   end
 
+  describe 'GET#index' do
+    before do
+      get :index
+    end
```

Comment:
one-line definition. also fix in the other places

Reply:
done